terminal/mac_osx.md

```diff
@@ -153,18 +167,20 @@ $ git config --global user.email "your@email.com"
 ```
 Now, you are ready to use Git. 
 
-#### Atom
-At the beginning of the course (week 1 - 6) we'll be using a text editor to write code. There are many editors out there we can use. [Sublime](http://www.sublimetext.com/) is one, [TextMate](https://macromates.com/) is another, but our editor of choice for this course is [Atom](https://atom.io). It is brought to us by the good people of GitHub and comes with over 3000 extension packages - all open sourced.
+#### Git Autocomplete
 
-*"Atom is a text editor that's modern, approachable, yet hackable to the core — a tool you can customize to do anything but also use productively without ever touching a config file."* - https://atom.io
-![Atom.io - Download for OS X](../images/atom_io_landing_page.png)
+Install bash-completion: `brew install brew install bash-completion` 
```

Comment:
You need to walk them through these steps more. They are not going to understand that you type this in the terminal or where to find that `.bash_profile`.